app/views/dashboard/index.html.slim

```diff
@@ -1,11 +1,12 @@
 h1 Cluster Status
-.row
-  .col-xs-12.discovery-control
-    span#unassigned_count
-      - if (unassigned = @unassigned_minions.count) > 0
+- if (unassigned = @unassigned_minions.count) > 0
+  .row
+    .col-xs-12.discovery-control
+      span#unassigned_count
         = unassigned
-        strong  new
-        | nodes are available but have not been added to the cluster yet
+          strong  new
+          | nodes are available but have not been added to the cluster yet
+= link_to "Download kubectl config", root_path, class: "btn btn-default btn-table pull-right"
```

Comment:
yes, @ereslibre needs to pass that one to me if it's already there. or we'll just add it later